[dynet] random seed: 3572390394
[dynet] allocating memory: 512MB
[dynet] memory allocation done.
/opt/conda/envs/xfactr/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3441: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/opt/conda/envs/xfactr/lib/python3.7/site-packages/numpy/core/_methods.py:189: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
load data
load model
acc per fact 0/0=0.0000	acc per relation nan	avg iter nan	num_max_mask 0
